scripts/1.TCGA-MLexample.py

```diff
@@ -72,84 +75,96 @@
         urllib.request.urlretrieve(url, path)
 
 
-# In[7]:
+# In[8]:
 
 get_ipython().run_cell_magic('time', '', "path = os.path.join('data', 'expression.tsv.bz2')\nX = pd.read_table(path, index_col=0)")
 
 
-# In[8]:
+# In[9]:
 
 get_ipython().run_cell_magic('time', '', "path = os.path.join('data', 'mutation-matrix.tsv.bz2')\nY = pd.read_table(path, index_col=0)")
 
 
-# ## Process Mutation Matrix
-
-# In[9]:
+# In[10]:
 
 y = Y[GENE]
 
 
-# In[10]:
+# In[11]:
 
 # The Series now holds TP53 Mutation Status for each Sample
 y.head(6)
 
 
-# In[11]:
+# In[12]:
 
 # Here are the percentage of tumors with NF1
 y.value_counts(True)
 
 
 # ## Set aside 10% of the data for testing
 
-# In[12]:
+# In[13]:
 
 # Typically, this can only be done where the number of mutations is large enough
 X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.1, random_state=0)
 'Size: {:,} features, {:,} training samples, {:,} testing samples'.format(len(X.index), len(X_train), len(X_test))
 
 
-# ## Feature transformation: Standardization
+# ## Median absolute deviation feature selection
 
-# In[13]:
+# In[14]:
 
-def scale_df(df):
-    """Standardize colummns by scaling to z-scores"""
-    scaled_matrix = preprocessing.scale(df)
-    return pd.DataFrame(scaled_matrix, index=df.index, columns=df.columns)
+def fs_mad(x, y):
+    mad = lambda a: np.median(np.absolute(a - np.median(a)))
```

Comment:
Even though MAD is simple, I prefer not to reimplement [`statsmodels.robust.scale.mad`](http://statsmodels.sourceforge.net/devel/generated/statsmodels.robust.scale.mad.html). `statsmodels=0.6.1` [is available](https://github.com/cognoma/machine-learning/pull/15/files#diff-b7cb75ca59993a879e29bed8cd8469b8R11) in our environment in #15.

Also make sure to add a docstring explaining what the function does.

Are you returning fake p-values? Can you return a vector of None or nan instead?